app/src/reducer/itemReducer.js

```diff
@@ -0,0 +1,8 @@
+export default function itemReducer(state = [], action) {
```

Comment:
File name example: container "Test" => reducer test.js and import into reducer.js , rootReducer